server/src/main/java/au/org/ala/names/ws/ALANameMatchingServiceConfiguration.java

```diff
@@ -33,13 +49,30 @@ public class ALANameMatchingServiceConfiguration extends Configuration {
      * </p>
      */
     public ALANameMatchingServiceConfiguration()  {
-        this.swagger.setTitle("ALA Namematching API");
-        this.swagger.setDescription("A taxonomy service that maps scientific name queries onto taxon concepts");
-        this.swagger.setContactUrl("https://ala.org.au");
-        this.swagger.setContactEmail("support@ala.org.au");
-        this.swagger.setResourcePackage("au.org.ala.names.ws.api,au.org.ala.names.ws.client,au.org.ala.names.ws.resources");
-        this.swagger.setLicense("Mozilla Public Licence 1.1");
-        this.swagger.setVersion("1.8-SNAPSHOT");
-        this.swagger.getSwaggerViewConfiguration().setPageTitle("ALA Namematching API");
+        // swagger UI
+        this.swaggerBundleConfiguration.setTitle("ALA Namematching API");
+        this.swaggerBundleConfiguration.setDescription("A taxonomy service that maps scientific name queries onto taxon concepts");
+        this.swaggerBundleConfiguration.setContactUrl("https://ala.org.au");
+        this.swaggerBundleConfiguration.setContactEmail("support@ala.org.au");
+        this.swaggerBundleConfiguration.setResourcePackage("au.org.ala.names.ws.api,au.org.ala.names.ws.client,au.org.ala.names.ws.resources");
+        this.swaggerBundleConfiguration.setLicense("Mozilla Public Licence 1.1");
+        this.swaggerBundleConfiguration.setVersion("1.8-SNAPSHOT");
```

Comment:
Is this correct ?

Reply:
Looks right to me. It's just a renaming of the bundle for clarity.

Reply:
I specifically meant the "1.8-SNAPSHOT" bit